### DISCUSSION TITLE
differences between pacakges

### DISCUSSION THREAD

**Discussion by Joduai** [phhusson/treble_experimentations#2457] [Q&A] [answered]
howdy
I wanted to give a shot with treble GSI but I'm completely confused seeing so many packages.
I've searched for at least a basic description of all packages for over an hour and I'm at my starting point: haven't found any info. I still have no idea what's the difference between gapps and gogapps (obviously it's about how many g.apps are packed, but which one?) the vanilla and vndklite. FOSS ver. seems obvious but still got no idea which is the most suitable for me and my motorola?

Could anyone point me to a thread on GH, reddit, xda or any other place I could read about differences between treble packages? Seems my skills in forming precise questions for search engines are non existant and finding any usable info novadays outgrows me completely.

**Comment by phhusson** [comment, marked as answer]
I already answered you in the other thread...

> It doesn't depend on the brand, but on the actual device. Assuming you have the most widespread variant you need an arm64_ab image. You can take vndklite or not to your liking, that won't change anything. and then you have vanilla/gapps/floss, gapps comes with gapps, floss comes with some floss apps to be usable (a web browser, a map tool, a calculator, etc.), vanilla comes with quite literally no apps.

gogapps means Android Go gapps, which are much more lightweight.
